src/components/GenreSelect/GenreSelect.tsx

```diff
@@ -1,31 +1,48 @@
 import { useState } from 'react';
 import './GenreSelect.scss';
 
-import { Genre, Genres } from '../../models/genres.model';
+import { Genres } from '../../models/genres.model';
 import classNames from 'classnames';
 
 
 type ResultsFilterProps = {
-  genres: Genre[];
-  selectedGenre: Genres;
-  onSelect: (genre: Genres) => void;
+  /**
+   * The list of genres which will be rendered
+   */
+  genres: Genres[];
+  /**
+   * A preselected genre
+   */
+  selectedGenre?: Genres;
+  /**
+   * Callback for pass a selected genre to a parent
+   * @param genre
+   */
+  onSelect?: (genre: Genres) => void;
 }
 
-function GenreSelect(props: ResultsFilterProps) {
-  const [genres] = useState(props.genres);
-  const [selectedGenre, setSelectedGenre] = useState(props.selectedGenre);
+/**
+ * The UI component for selecting a genre from the list of genres
+ */
+function GenreSelect({
+  genres = [],
+  selectedGenre = Genres.ALL,
+  onSelect
+}: ResultsFilterProps) {
+  const [getGenres] = useState(genres);
```

Comment:
There is a naming convention about how should state be named.
Please follow approach:
`const [state, setState] = useState();`
`const [genres, setGenres] = useState();`
`const [selectedGenre, setSelectedGenre] = useState();`

Reply:
Fixed, kinda